RELEASING.md

```diff
@@ -9,12 +9,13 @@
 1. Tag the release: `git tag vVERSION`
 1. Push changes: `git push --tags`
 1. Build and publish:
+  ```bash
+  gem build project-name.gemspec
+  gem push project-name-*.gem
+  ```
 
-```bash
-gem build doorkeeper.gemspec
-gem push doorkeeper-*.gem
-```
-
+1. Add a new GitHub release, copying as message the corresponding `NEWS.md`
+   content. Sample URL: https://github.com/org/project-name/releases/new?tag=vVERSION
```

Comment:
Would it make sense to replace `org` with `thoughtbot`, since these templates are for our benefit, and `thoughtbot` will likely be the most common organization?